lib/plugins/usermanager/lang/it/lang.php

```diff
@@ -66,8 +66,6 @@
 $lang['add_fail']              = 'Aggiunta utente fallita';
 $lang['notify_ok']             = 'Email di notifica inviata';
 $lang['notify_fail']           = 'L\'email di notifica non può essere inviata';
-$lang['import_userlistcsv']    = 'File lista utente (CSV):';
```

Comment:
You are right!
I wasn't able to pull a translation because the translation tool gave me an error [*], so I re-translated and, maybe, I did a mistake!
Please, don't merge this! I will re-translate on more time.

[*]: actually, it always rejects a pull request if I have two translation pages open.